File: Client/src/log/logger.ts
import * as alt from "alt-client"

export function info(msg: string) {
    alt.log('~g~' + msg)
}

export function warn(msg: string) {
    alt.log('~y~' + msg)
}

export function debug(msg: string) {
    alt.log('~y~' + msg)
}

export function error(msg: string) {
    alt.log('~lr~' + msg)
}

File: Server/src/Log/Logger.cs
using AltV.Net;
using AltV.Net.ColoredConsole;

namespace MainResource.Log;

public static class Logger
{
    // 提示日志
    public static void Info(string msg)
    {
        var coloredMessage = new ColoredMessage();
        coloredMessage += TextColor.Green;
        coloredMessage += msg;
        Alt.LogColored(coloredMessage);
    }
    // 警告日志
    public static void Warn(string msg)
    {
        var coloredMessage = new ColoredMessage();
        coloredMessage += TextColor.Yellow;
        coloredMessage += msg;
        Alt.LogColored(coloredMessage);
    }
    // 错误日志
    public static void Error(string msg)
    {
        var coloredMessage = new ColoredMessage();
        coloredMessage += TextColor.Red;
        coloredMessage += msg;
        Alt.LogColored(coloredMessage);
    }
}